cmsplugin_filer_tests_shared/base.py

```diff
@@ -43,11 +52,13 @@ def create_plugin(self, page=None, language=None, **kwargs):
     def test_plugin_can_be_added(self):
         plugin = self.create_plugin()
         self.assertTrue(plugin is not None)
-        self.assertIn(plugin, self.page.placeholders.all()[0].get_plugins())
+        self.assertIn(plugin.cmsplugin_ptr,
+                      self.page.placeholders.all()[0].get_plugins())
 
     def test_plugin_does_not_breakes_page(self):
```

Comment:
``test_plugin_does_not_break_page`` is the next test, which also fetches the page. this one only checks if plugin is present in page placeholder.